src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Http/Module.ts

```diff
@@ -1,19 +1,20 @@
 import * as AdhCredentialsModule from "../User/CredentialsModule";
+import * as AdhMetaApiModule from "../MetaApi/Module";
 import * as AdhPreliminaryNamesModule from "../PreliminaryNames/Module";
 import * as AdhWebSocketModule from "../WebSocket/Module";
 
 import * as AdhCache from "./Cache";
 import * as AdhError from "./Error";
 import * as AdhHttp from "./Http";
-import * as AdhMetaApi from "./MetaApi";
 
 
 export var moduleName = "adhHttp";
 
-export var register = (angular, config, metaApi) => {
+export var register = (angular, config) => {
     angular
         .module(moduleName, [
             AdhCredentialsModule.moduleName,
+            AdhMetaApiModule.moduleName,
```

Comment:
sry, I'm a little slow: Do we need the metaApi.moduleName here because `"adhMetaApi"` is used in the `.service` registration below?